Sources/MongoSwift/MongoCollection.swift

```diff
@@ -84,11 +95,17 @@ public class MongoCollection<T: Codable> {
         self._collection = nil
     }
 
-    /// Drops this collection from its parent database.
-    /// - Throws:
-    ///   - `ServerError.commandError` if an error occurs that prevents the command from executing.
-    public func drop(session: ClientSession? = nil) throws {
-        let operation = DropCollectionOperation(collection: self, session: session)
+    /**
+    *   Drops this collection from its parent database.
+    * - Parameters:
+    *   - session: An optional `ClientSession` to use when executing this command
```

Comment:
nit: switch order of session and options here to match order in the function signature